test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -84,9 +88,15 @@ public EdgeConfiguration Build()
                         ["properties.desired"] = module.DesiredProperties
                     };
                 }
+
+                if (module.Name == "$edgeHub")
```

Comment:
use string.Equals with comparison option

Reply:
I made this change